### PR TITLE
Update ReadME.md to correct discord link

### PR DESCRIPTION
**Pull request by PhantomG27249**

### CODE REVIEW

**Comment by duplexsystem** [comment]
you included the filtering stuff

**Comment by PhantomG27249** [comment]
@budgidiere ah sorry just deleted them, should just be the read me now

**Comment by MrIvanPlays** [comment]
fix the diff

File: PATCHES.md [hunk 1]
```diff
@@ -1,122 +0,0 @@
...
-| server |  lithium MixinBox      | JellySquid |  |
-| server |  lithium MixinChunkSection      | JellySquid |  |
-| server |  lithium MixinDirection      | JellySquid |  |
-| server |  lithium MixinGoalSelector      | JellySquid |  |
-| server |  lithium MixinLandPathNodeMaker      | JellySquid |  |
-| server |  lithium VoxelShapesMixin      | JellySquid | Ivan Pekov |
-| server |  lithium collision optimizations      | JellySquid | Ivan Pekov |
-| server |  lithium enum_values      | JellySquid |  |
-| server |  lithium-MixinChunkGeneratorAbstract      | JellySquid | tr7zw |
-| server |  lithium-MixinDataWatcher      | JellySquid | tr7zw |
```
Comment (toinouH) [inline PATCHES.md]: We do not want to delete that.

**Review by MrIvanPlays — APPROVED** [review]

**Pull request merged**